src/main/java/InputReader.java

```diff
@@ -0,0 +1,53 @@
+/**
+ *
+ */
+public class InputReader {
+
+    public InputReader() {
+    }
+
+    /**
+     * Uses Regex to replace all instances of non digit characters to be parsed as int
+     *
+     * @param input
+     * @return int for the task index
+     */
```

Comment:
Good use of Javadoc to make your code much easier to follow!

Reply:
Thank you!